### DISCUSSION TITLE
Problema Invio Fattura a SDI - Campo Provincia Esteso

### DISCUSSION THREAD

**Discussion by Dreamerscommunity** [fattureincloud/api#470] [Q&A] [answered]
Ciao a tutti

Da un po' di giorni ho completato l'integrazione tra STRIPE e FIC con Zapier e ho già emesso diverse fatture in automatico senza problemi

Oggi però su una fattura riscontro questo problema.

La fattura è stata creata MA non inviata allo SDI. Credo il motivo sia che l'utente ha scritto "Verona" anzichè "VR", che è quello che Fatture in Cloud si aspettava. Ma come è possibile che l'utente sia riuscito a scrivere questo valore?

![image](https://github.com/user-attachments/assets/b463a257-b95b-4037-a542-c247b70563cd)

Come potrei correggere il problema?

Grazie mille

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Dreamerscommunity 
mi spiace, ma il valore che viene inserito all'interno della richiesta non dipende da noi, in quanto immagino sia stato inserito in Zapier a partire da una sorgente a monte. Dovresti verificare nel punto in cui l'utente ha inserito il campo provincia, presumibilmente in Stripe oppure nel vostro sito.

Grazie
Mauro

**Comment by Dreamerscommunity** [comment]
Buongiorno @valmoz riprendo questo thread dopo un po' di tempo per chiederti se c'è un modo lato vostro per accettare la provincia sia in sigla che per esteso, in modo che la fattura venga inviata automaticamente allo SDI.

Io ho costruito il flusso su Zaple e ho pensato ad uno step con una lookup table che faccia la riformattazione della provincia se scritta per estesa ma non saprei se fidarmi. Se ci sono soluzioni più semplici le valuterei volentieri. Grazie!

Marco
